pkg/subctl/cmd/join.go

```diff
@@ -80,8 +80,11 @@ const (
 var joinCmd = &cobra.Command{
 	Use:   "join",
 	Short: "connect a cluster to an existing broker",
-	Args:  cobra.ExactArgs(1), // exactly one, the broker data file
+	Args: cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			panic("broker-info.subm file generated by subctl deploy-broker not passed")
```

Comment:
`panic` still results in a rather unfriendly stacktrace:

```
panic: broker-info.subm file generated by subctl deploy-broker not passed

goroutine 1 [running]:
github.com/submariner-io/submariner-operator/pkg/subctl/cmd.glob..func2(0x23d2180, 0x2415970, 0x0, 0x0)
	/go/src/github.com/submariner-io/submariner-operator/pkg/subctl/cmd/join.go:86 +0x18f
github.com/spf13/cobra.(*Command).execute(0x23d2180, 0x2415970, 0x0, 0x0, 0x23d2180, 0x2415970)
	/go/src/github.com/submariner-io/submariner-operator/vendor/github.com/spf13/cobra/command.go:830 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x23d2400, 0x1255ce3, 0xc0006a7f88, 0xc00003c118)
	/go/src/github.com/submariner-io/submariner-operator/vendor/github.com/spf13/cobra/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
	/go/src/github.com/submariner-io/submariner-operator/vendor/github.com/spf13/cobra/command.go:864
github.com/submariner-io/submariner-operator/pkg/subctl/cmd.Execute(...)
	/go/src/github.com/submariner-io/submariner-operator/pkg/subctl/cmd/root.go:52
main.main()
	/go/src/github.com/submariner-io/submariner-operator/pkg/subctl/main.go:22 +0x2f
```

Please adapt `exitOnError()` (without passing an error) instead.